django-stubs/contrib/gis/db/models/fields.pyi

```diff
@@ -108,7 +108,9 @@ class GeometryField(BaseSpatialField[_ST, _GT]):
         validators: Iterable[_ValidatorCallable] = ...,
         error_messages: _ErrorMessagesMapping | None = ...,
     ) -> None: ...
-    def formfield(self, **kwargs: Any) -> Any: ...  # type: ignore[override]
+    def formfield(  # type: ignore[override]
+        self, form_class: type[forms.GeometryField] | None = ..., geom_type: str = ..., srid: Any = ..., **kwargs: Any
+    ) -> forms.GeometryField: ...
```

Comment:
No need to qualify these types, they are defined in this very file.

```suggestion
        self, form_class: type[GeometryField] | None = ..., geom_type: str = ..., srid: Any = ..., **kwargs: Any
    ) -> GeometryField: ...
```

Reply:
@intgr Likewise, these are not the same types.

`forms.GeometryField` is a form field. The local `GeometryField` is a model field.